reana/reana_dev/cluster.py

```diff
@@ -376,10 +376,26 @@ def cluster_unpause():
     run_command(cmd, "reana")
 
 
+@click.option(
+    "-m",
+    "--mount",
+    "mounts",
+    multiple=True,
+    help="Which directories are used to mount the cluster nodes? [local_path:cluster_node_path]",
```

Comment:
Please change the help message to `Which local path directories are to be deleted? [local_path:cluster_node_path]"` in order to better indicate that these will get deleted. (danger!)